sdk/identity/azure-identity/azure/identity/_credentials/environment.py

```diff
@@ -14,14 +14,12 @@
 from .client_secret import ClientSecretCredential
 from .user_password import UsernamePasswordCredential
 
-
-if TYPE_CHECKING:
-    EnvironmentCredentialTypes = Union["CertificateCredential", "ClientSecretCredential", "UsernamePasswordCredential"]
+EnvironmentCredentialTypes = Union["CertificateCredential", "ClientSecretCredential", "UsernamePasswordCredential"]
```

Comment:
I don't think we need forward refs here - are you seeing a weird error?

Reply:
I used this to make sure 

self._credential: Optional[EnvironmentCredentialTypes] = None

isn't too long. :)

Reply:
Yeah the type alias is great! I was referring to the Credential types as strings

Reply:
Sorry I mis-understood.

Fixed.